Nimble/Wrappers/ObjCMatcher.swift

```diff
@@ -118,7 +44,7 @@ typealias FullMatcherBlock = (actualExpression: Expression<NSObject>, failureMes
     }
 
     public func matches(actualBlock: () -> NSObject!, failureMessage: FailureMessage, location: SourceLocation) -> Bool {
-        let expr = Expression(expression: actualBlock, location: SourceLocation())
+        let expr = Expression(expression: actualBlock, location: location)
```

Comment:
Not that I could tell, although it's weird either way. Nimble's Objective-C matcher API probably needs some reworking for this. I'll need to take another pass to see if it's worth removing use of `location` in `MatcherBlock` and `FullMatcherBlock`.